src/index.js

```diff
@@ -0,0 +1,57 @@
+import './index.css';
+// Uncomment to get a new id
+// import setNewGame from './modules/game.js';
+import addNewUser from './modules/user.js';
+import getAllUsers from './modules/getusers.js';
+
+// console.log(setNewGame());
+// Invoke function to post a new game
```

Comment:
- [ ] Kindly delete commented codes to follow the best practices. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)